src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.Tests/DependencyAddUpdateTests.cs

```diff
@@ -447,6 +447,28 @@ public void UpdateArcadeDependency2()
             });
         }
 
+        /// <summary>
+        /// Update the arcade dependency to a new version, with an associated global.json update.
+        /// </summary>
+        [Fact]
+        public void UpdateArcadeDependencyWithSdkUpdate()
```

Comment:
Can we also tell the test to update the .net sdk? not sure how much work it'd be to set up.

Reply:
aaand I just saw the test for 10.1.blah. Thanks!

Reply:
https://github.com/dotnet/arcade-services/pull/1157/files/99ac5e3bb3ad8dad909df748b60fb32afbf31632#diff-4d6e937f33124cd7921ab82725c84032R467

That's what this is doing.

Reply:
Yep. I noticed that after posting the initial comment. sorry for the noise.